### ISSUE TITLE
Pylint do not raise a FATAL when the toml configuration is impossible to parse

### ISSUE THREAD

**Issue by Pierre-Sassoulas** [PyCQA/pylint#4204]
This is a follow-up to https://github.com/PyCQA/pylint/pull/4176

Pylint does:
```
qlyoung@nicobar ~> pylint ./frrbot.py
Failed to load 'pyproject.toml': This float doesn't have a leading digit (line 28 column 1 char 585)
Failed to load 'pyproject.toml': This float doesn't have a leading digit (line 28 column 1 char 585)
```
But the expected output is a FATAL message with an exit output 1. It should be done by adding a FATAL error message in the Pylinter class, maybe F0003 ``pyproject-toml-parse-error``. This means a refactoring of the configuration.

**Comment by Pierre-Sassoulas** [comment]
Adding to 2.14 because I think it's been fixed for some time now.

**Comment by DanielNoord** [comment]
Added to the `argparse` project. Should be trivial to fix this and add a warning about it.

**Comment by DanielNoord** [comment]
We emit `config-parse-error` now. So I"m closing this!
